src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -409,7 +410,7 @@ public void mergeAlignment(final File referenceFasta) {
                         final SAMRecord matePrimary = isRead1 ? r2Primary : r1Primary;
 
                         for (final SAMRecord supp : supplementals) {
-                            final SAMRecord out = clone(sourceRec);
+                            final SAMRecord out = sourceRec.deepCopy();
```

Comment:
@yfarjoun You think that supplementals can be on different strands even if that violates the SAM spec? Also it doesn't make intuitive sense to me how a supplemental alignment could biologically come from different strands, whereas secondary alignments are mutually exclusive biologically, so it's clear how they can be on different strands.